ecosystem/sep-0010.md

```diff
@@ -14,6 +14,8 @@ Version 2.0.0
 
 This SEP defines the standard way for clients such as wallets or exchanges to create authenticated web sessions on behalf of a user who holds a Stellar account. A wallet may want to authenticate with any web service which requires a Stellar account ownership verification, for example, to upload KYC information to an anchor in an authenticated way as described in [SEP-12](sep-0012.md).
 
+This SEP also supports clients who control [pooled accounts](#pooled-accounts) containing funds owned by _many_ users. These clients are _custodians_ of their users' Stellar accounts and possess the secret keys for those accounts.
```

Comment:
I think we should avoid the use custodian. That term has specific legal meaning which is outside the scope of this technical spec. I also think we're being unnecessarily prescriptive to discuss secret keys of accounts.
```suggestion
This SEP also supports clients who control [pooled accounts](#pooled-accounts) containing funds owned by _many_ users.
```